rustfmt.toml

```diff
@@ -1,10 +1,11 @@
 edition = "2021"
-version = "Two"
 reorder_imports = true
-imports_granularity = "Item"
-group_imports = "StdExternalCrate"
-where_single_line = true
-trailing_comma = "Vertical"
-overflow_delimited_expr = true
-format_code_in_doc_comments = true
-normalize_comments = true
+
+# version = "Two"
```

Comment:
What about just removing these style settings?

Reply:
I expect to enable them back once got stabilized. It looks good to me to remove them.

Reply:
I see.

I think you can keep them but leave a comment about adding them back once stabilized. Otherwise, we will forget why they are commented out.